data/the emerald sword.txt

```diff
@@ -0,0 +1,796 @@
+# Copyright (c) 2020 by Fernando S. Sáez
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+mission "The Box 1"
+	landing
+	name "The Sheragi bunker"
+	description "Visit Albert Foster on Zug to see what he discovered."
+	source
+		near "Zubeneschamali" 1 100
+	destination Zug
+	
+	to offer
+		has "event: rim archaeology results"
+		has "First Contact: Unfettered: declined"
+		has "main plot completed"
+		random < 4
+	
+	on offer
+		conversation
+			`You receive a message from Albert Foster, the archaeologist who you helped uncover the mysterious Sheragi ruins on Zug. He seems to be very excited.`
+			`	"<first>! It's been a while, huh? Remember the Sheragi dig site? I've just found something incredible here. Please, come visit me on Zug. I don't think I can trust anyone else with this."`
+				accept
+
+mission "The Box 2"
+	landing
+	name "The mysterious box"
+	description "Take the mysterious box to a trusted scientist on <destination>."
+	source "Zug"
+	destination "Vinci"
+	
+	to offer
+		has "The Box 1: done"
+	
+	on offer
+		conversation
```

Comment:
This conversation is so long that it actually goes off of my screen, which is somewhat problematic. I assume this is due to the fact that some parts of the code have been moved around incorrectly. I am attaching a screenshot at [this](https://drive.google.com/file/d/1RS7D1ERscLF0CZdbgpTRWaBoapFitTf4/view?usp=sharing) link so that you can see what I mean.